task_geo/data_sources/covid_cases/all_countries/all_countries.py

```diff
@@ -0,0 +1,85 @@
+import pandas as pd
+import numpy as np
+import time
+
+
+def all_country_datasource_connector(*args, **kwargs):
+  '''
+    Input:  None
+    Output: Return raw dataframe of all countries
+  '''
+  df = pd.read_csv("https://coronadatascraper.com/timeseries.csv")
+  
+  return df
+
+def all_country_datasource_formatter(df):
+  '''
+    Input:  Raw data from connector method
+    Output: Return list of dataframe of all countries
+  '''
+
+  df_list = []
+
+  #Remove rows where country is not provided
+  df = df[df['country'].notna()]
+  #df = df[df['state'].notna()]
+
+  #Fill NULL values with 0 for features - cases, deaths, recovered, active, growthFactor
```

Comment:
Avoid filling in missing values with 0 here, could be misleading!

Reply:
Thanks a lot. I have removed these lines.